src/api/notification/content-types/notification/schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "kind": "collectionType",
+  "collectionName": "notifications",
+  "info": {
+    "singularName": "notification",
+    "pluralName": "notifications",
+    "displayName": "Notification",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
```

Comment:
should we have a field consumed? or you want to DELETE when consumed?

Reply:
We decided to use `lastConsumedNotificationDate` value